build.sbt

```diff
@@ -134,6 +134,17 @@ lazy val `play-json` = crossProject(JVMPlatform, JSPlatform).crossType(CrossType
         case Some((2, 13)) => Seq()
         case _ => Seq(compilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full))
       }),
+    // Adds a `src/main/scala-2.13+` source directory for Scala 2.13 and newer
```

Comment:
This is conflicting with https://github.com/playframework/play-json/pull/230.

Reply:
fixed!